CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.11.4
+
+### BPMN
+
+* `FIX`: gracefully handle incompatible properties providers
+* `FIX`: re-use ID of a copied element if available
+
```

Comment:
Should we not add this fix as well:
https://github.com/camunda/camunda-bpmn-js/commit/94845618d076601e17733c82a01767a7e2e0f024
?

Reply:
ah, sorry missed that